### ISSUE TITLE
Can't export note to PDF in elementary OS 5

### ISSUE THREAD

**Issue by joaovitor123jv** [Philip-Scott/Notes-up#336]
## Expected Behavior
Export a valid PDF file from a note written in markdown.

## Current Behavior
After writing a note and checking that it is rendered in-app, when I click "export to PDF" 
and select a file, the file is created without content.

The generated file has 0 Bytes of content, neither the magic number.

## Possible Solution
Can it be a missing dependency on the build script?

## Steps to Reproduce (for bugs)
1. Install `Notes-Up` from app-center on elementary OS 5
2. Write a note in markdown
3. Click the "gear" icon
4. Click "Export as PDF"
5. Select a place and name the file as `file.pdf`
6. Click `save` button in file explorer
7. Try to open the created file with a PDF reader

## Context
I cannot export the note as PDF, I think it's a key function of this app

Debug:
```
{12:04}[2.6.0]~/ ✗ ➭ G_MESSAGES_DEBUG=all com.github.philip-scott.notes-up
[INFO 12:04:16.104777] Application.vala:155: Notes-Up version: 2.0.0
[INFO 12:04:16.104827] Application.vala:157: Kernel version: 4.15.0-65-generic
(Notes-Up:10153): Gtk-DEBUG: 12:04:16.150: Connecting to session manager
Previous page set
Previous page set
{12:05}[2.6.0]~/ ✗ ➭ 
```

## Screenshots of the Problem
![image](https://user-images.githubusercontent.com/12112729/68034381-4f7bef80-fca0-11e9-8450-100ba5d542ca.png)
![image](https://user-images.githubusercontent.com/12112729/68034419-63bfec80-fca0-11e9-88b6-ea0b2cff8a37.png)
![image](https://user-images.githubusercontent.com/12112729/68034290-2e1b0380-fca0-11e9-941c-82e7c0666205.png)

## Your Environment (for bugs)
```
{12:10}[2.6.0]~ ➭ uname -a && cat /etc/os-release
Linux joaovitor-elementary 4.15.0-65-generic #74-Ubuntu SMP Tue Sep 17 17:06:04 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
NAME="elementary OS"
VERSION="5.0 Juno"
ID=elementary
ID_LIKE=ubuntu
PRETTY_NAME="elementary OS 5.0 Juno"
LOGO=distributor-logo
VERSION_ID="5.0"
HOME_URL="https://elementary.io/"
SUPPORT_URL="https://elementary.io/support"
BUG_REPORT_URL="https://github.com/elementary/appcenter/issues/new"
PRIVACY_POLICY_URL="https://elementary.io/privacy-policy"
VERSION_CODENAME=juno
UBUNTU_CODENAME=bionic
```

- Notes-Up Version:
![image](https://user-images.githubusercontent.com/12112729/68034842-36277300-fca1-11e9-892f-d25b05d277d7.png)

- OS Info: `Linux joaovitor-elementary 4.15.0-65-generic #74-Ubuntu SMP Tue Sep 17 17:06:04 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux`

**Comment by Philip-Scott** [comment]
Are you using the app in an environment with a different language? :) If so the answer for how to solve it for you and everyone else is over at #1

**Comment by joaovitor123jv** [comment]
Yes, I'm using the app in Brazilian Portuguese. Later I'll try to create a translation and compile it to check if I can fix it. 

Thank you!
